app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiVerifyUsedInLinkCategoryGridTest.xml

```diff
@@ -22,15 +19,18 @@
             <group value="media_gallery_ui"/>
         </annotations>
         <before>
-            <createData entity="SimpleSubCategory" stepKey="category"/>
+            <createData entity="TestSubCategory" stepKey="category"/>
             <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsAdmin"/>
+            <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushCache">
```

Comment:
Please, avoid flushing the cache whenever it's possible. This action is slow itself and slows down the consecutive tests significantly. I would strongly recommend clearing the proper cache tags instead

https://github.com/magento/magento2/blob/e9190a5bb2c4aede2a752e94a3e87dcf9647b604/app/code/Magento/Backend/Test/Mftf/ActionGroup/CliCacheCleanActionGroup.xml#L11